lib/project_types/script/layers/infrastructure/script_project_repository.rb

```diff
@@ -36,6 +36,7 @@ def create(script_name:, extension_point_type:, language:)
             organization_id: nil,
             extension_point_type: extension_point_type,
             script_name: script_name,
```

Comment:
Nit: This can be fixed in another PR, but should this say `title` instead of `script_name` to be consistent with the rest of the system?